assets/scripts/dialogs/AboutDialog.jsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * About Streetmix (dialog box)
+ *
+ * Handles the "About" dialog box.
+ * Instantiates an instance of Dialog
+ * Exports nothing
+ *
+ */
+import React from 'react'
+import Dialog from './Dialog.jsx'
```

Comment:
Good catch. We're going to stick to not having a file extension. The [AirBnb React styleguide](https://github.com/airbnb/javascript/tree/master/react) does not explicitly require it, but implies it in their example.